syntax = "proto2";
package spider;

message pingpong_req {
	required int32 num = 1;	
}
